myuw/views/api/instructor_schedule.py

```diff
@@ -68,8 +83,8 @@ def set_course_resources(section_data, section):
 
     if section.class_website_url:
         t = ThreadWithResponse(
```

Comment:
If you're going to use threadwithresponse, I don't think it should track the response and also modify a passed in dictionary.  i'd be fine w/ either a more complex value returned by the function, or moving away from threadwithresponse